<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAdmin</class>
 <widget class="QWidget" name="AddAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;
border: 1px solid #ccc;
border-radius: 5px;
background-image: url(:/ing/AddAdminImg.png);
background-repeat:no-repeat;
background-position:center;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#label_11 {
background:none;
    background-color: #f7f7f7; /* Light grey background */
    color: #333333; /* Dark grey text */
    border: 1px solid #d9d9d9; /* Light grey border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Padding around the text */
    font-size: 20px; /* Font size appropriate for the label size */
    font-weight: bold; /* Bold text */
    text-align: center; /* Center the text */
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Subtle shadow effect */
    margin-bottom: 10px; /* Space below the label */
    letter-spacing: 1px; /* Spacing between letters */
    height: 30px; /* Explicit height for the label */
}
</string>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>481</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;
background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>120</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
border:none;
</string>
         </property>
         <property name="text">
          <string>Admin ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>180</y>
           <width>120</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
         </property>
         <property name="text">
          <string>Frist Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>300</y>
           <width>120</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
         </property>
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="AI_AdminID">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>70</y>
           <width>250</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="AI_FirstName">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>180</y>
           <width>250</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);
}</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="AI_LastName">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>300</y>
           <width>250</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);
}</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>70</y>
           <width>120</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="AI_Username">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>70</y>
           <width>250</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>180</y>
           <width>120</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="AI_Password">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>180</y>
           <width>250</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);
}</string>
         </property>
        </widget>
        <widget class="QPushButton" name="AI_Update">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>430</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background:none;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Cencel">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>430</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background:none;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButton">
         <property name="geometry">
          <rect>
           <x>950</x>
           <y>180</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
qproperty-icon:url(:/ing/pngwing.com.png);
qproperty-iconSize: 35px 35px;
border: 0px solid
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
